workspace-manager-client/src/main/resources/swaggercodegen/libraries/jersey2/api.mustache

```diff
@@ -0,0 +1,133 @@
+{{!--
+  Copied from the https://github.com/swagger-api/swagger-codegen-generators tagged release 1.0.24:
+  https://github.com/swagger-api/swagger-codegen-generators/blob/v1.0.24/src/main/resources/handlebars/Java/libraries/jersey2/api.mustache
+
+  This file may include Terra-specific modifications to the Swagger-codegen generated API client
+  code. This file (api.mustache) is responsible for generating each typed API class, e.g.
+  bio.terra.*.api.FooApi.
+
+  All Terra changes or additions should be surrounded by comment blocks of the form:
+
+  {{! Begin Terra changes }}
```

Comment:
is the trick to use a partial for the terra snippets applicable here?

Reply:
I could go either way, but I'm inclined to leave as-is for simplicity's sake (fewer files and less indirection). It's likely that we'll ultimately want to manage & publish these template overrides from terra-common-lib, so that will provide another chance to assess the ideal structure.